### PR TITLE
Update Xamarin release notes

### PR DESCRIPTION
**Pull request by spoorthipujariadobe**

### CODE REVIEW

File: release-notes/README.md [hunk 1]
```diff
@@ -4,7 +4,36 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 # Release notes
 
-## March 3, 2021
```
Comment (emdobrin) [inline release-notes/README.md:7]: We should keep March 3 for the Android Analytics and add a new section for the Xamarin release on March 4. 
Also, could we use a similar format as we have for Unity/Flutter releases: https://aep-sdks.gitbook.io/docs/release-notes#acp-unity-packages

```
## March 4, 2022
### ACP Xamarin frameworks
Updated the Android and iOS native libraries for the following Xamarin frameworks:
* Android Analytics framework v1.0.1 updated to v1.2.10 (Android) and v2.5.1 (iOS)
* ...

File: release-notes/README.md [hunk 2]
```diff
@@ -4,7 +4,23 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
...
```
Comment (emdobrin) [inline release-notes/README.md:7]: Nice catch, leaving in the past 🙈 
Can you please update the year for this one too https://github.com/Adobe-Marketing-Cloud/aep-sdks-documentation/blob/master/using-mobile-extensions/adobe-analytics/adobe-analytics-release-notes.md#march-3-2021 ?

File: release-notes/README.md [hunk 3]
```diff
@@ -4,7 +4,23 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
...
+## March 4, 2022
+
+### ACP Xamarin frameworks
+
+Updated the Android and iOS native libraries for the [ACPCore](https://github.com/adobe/xamarin-acpcore) and [ACPAnalytics](https://github.com/adobe/xamarin-acpanalytics) Xamarin frameworks:
+
+* [Xamarin Core Android 1.1.0](https://www.nuget.org/packages/Adobe.ACPCore.Android/1.1.0) : Android Core updated to 1.10.0
```
Comment (emdobrin) [inline release-notes/README.md:13]: nit: can you remove the extra space before column for all occurrences below, e.g:
[Xamarin Core Android 1.1.0](https://www.nuget.org/packages/Adobe.ACPCore.Android/1.1.0): Android Core updated to 1.10.0

**Review by emdobrin — APPROVED** [review]
Looks great, thanks @spoorthipujariadobe

File: release-notes/README.md (hunk 2 above)
Comment (spoorthipujariadobe) [reply release-notes/README.md:7]: Have updated it already :)

File: release-notes/README.md (hunk 3 above)
Comment (spoorthipujariadobe) [reply release-notes/README.md:13]: Removing the space was messing up the link when I tried it in Xcode editor, let me give it a try

**Pull request merged**